### PR TITLE
Open Embedded base image

### PR DESCRIPTION
**Pull request by petrosagg**
This PR adds all the infrastructure needed to build a `supervisor-base` image that is built using open embedded, instead of using a Dockerfile based on alpine or debian

---
This change is [Reviewable](https://reviewable.io/reviews/resin-io/resin-supervisor/198)

### CODE REVIEW

**Comment by petrosagg** [comment]
@floion @nghiant2710 @pcarranzav

**Comment by petrosagg** [comment]
@agherzan

File: base-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:15.04
+
+# Install the following utilities (required by openembedded)
```
Comment (agherzan) [inline base-image/Dockerfile]: Would be nice to have a link to the documentation where the deps are stated: updating yocto would be easier in this way,

File: base-image/automation/jenkins-build.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ docker run --rm \
 	-v `pwd`/dest:/dest \
 	supervisor-base-builder
 if [ -f dest/rootfs.tar.gz ]; then
-	cat dest/rootfs.tar.gz | docker import - $REPO:$date
```
Comment (agherzan) [inline base-image/automation/jenkins-build.sh]: This is not related to the commit log, is it?

File: base-image/automation/jenkins-build.sh [hunk 3]
```diff
@@ -5,21 +5,43 @@ set -o pipefail
...
-if [ -f dest/rootfs.tar.gz ]; then
-	docker import dest/rootfs.tar.gz $REPO:$date
-	docker tag -f $REPO:$date $REPO:latest
-	docker push $REPO
-else
-	echo "rootfs is missing!"
-	exit 1
-fi
+for machine in $MACHINE_LIST; do
+	case "$device" in
```
Comment (agherzan) [inline base-image/automation/jenkins-build.sh]: type: machine.

**Comment by agherzan** [comment]
Small comments but generally it looks good to me.

**Comment by petrosagg** [comment]
Review status: 0 of 17 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[base-image/Dockerfile, line 3 at r3](https://reviewable.io:443/reviews/resin-io/resin-supervisor/198#-KM3PN-gvJf3iCzoxt4x-r3-3:-KRaKRVkbC3QwvdvIYSv:b-896fix) ([raw file](https://github.com/resin-io/resin-supervisor/blob/50e512f0792b01a1a64ca5e42fe36dab9e0d8318/base-image/Dockerfile#L3)):*
<details><summary><i>Previously, agherzan (Andrei Gherzan) wrote…</i></summary>
> Would be nice to have a link to the documentation where the deps are stated: updating yocto would be easier in this way,
</details>
Done.

---

*[base-image/automation/jenkins-build.sh, line 17 at r4](https://reviewable.io:443/reviews/resin-io/resin-supervisor/198#-KM3PN-gvJf3iCzoxt4y-r4-17:-KRaKTOIF-Tg8ymovoYn:b-896fix) ([raw file](https://github.com/resin-io/resin-supervisor/blob/70b81768de01a321d44cc23e549a03f15f8bd61a/base-image/automation/jenkins-build.sh#L17)):*
<details><summary><i>Previously, agherzan (Andrei Gherzan) wrote…</i></summary>
> typo: machine.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/resin-io/resin-supervisor/198)*
